src/components/DeviceInfo.vue

```diff
@@ -16,30 +16,24 @@
 
 <script>
 
-import axios from "axios";
+import { mapState} from 'vuex';
+
 
 export default {
     name: "DeviceInfo",
-    data() {
-        return {
-            devices: null
-        }
-    },
-    methods: {
-        getSensors() {
-            axios.get(`${this.$VUE_APP_BACKEND_BASE_URL}/devices`)
-            .then((response) => {
-                
-                this.devices = response.data;
-            })
-            .catch((error) => {
-                console.log(error);
-            })
-        
-        }
-    },
-    created: function(){
-        this.getSensors();
+
+
+    created(){
+        this.$store.dispatch('getAllSensors');
     },
+
+    computed:{
+        ...mapState([
+            'devicelist'
+        ]),
```

Comment:
`mapGetters` is better than `mapState` for reactive updates but I will make new issue for this so this doesn't need to be changed.